python_src/src/lamp_py/ingestion/convert_gtfs.py

```diff
@@ -97,3 +52,70 @@ def convert(self) -> List[str]:
             )
 
         return error_files + archive_files
+
+    def process_schedule(self, filename: str) -> None:
+        """
+        convert a schedule gtfs zip file into tables. the zip file is
+        essentially a small database with each contained file (outside of feed
+        info) acting as its own table. info on the gtfs scheduling standard can
+        be found at http://gtfs.org/schedule/
+        """
+        # s3 objects are read directly from s3 as a stream of bytes
+        filelike_input: Union[str, IO[bytes]] = filename
+
+        # the last modified date is pulled from s3 and we use it as our version
+        # key to link all of the table data together
+        last_modified = 0
```

Comment:
I think we should take this opportunity to stop using the last modified date for zip file itself as the `timestamp` to link table data together. 

`ZipInfo.date_time` provides the last modified date of the files in the archive. Using this value from the `feed_info.text` file inside of the archive seems like a good option, that won't be altered by moving the archive file itself around S3.

Reply:
I think the only other viable option would be to extract the datetime from the `feed_version` field inside of the `feed_info.txt` file.